content/blog/2020-08-28-api.Rmd

```diff
@@ -397,15 +400,22 @@ According to the `r blogdown::shortcode_html("apireflink", "api/covidcast-signal
 this is the `fb-survey` data source's `smoothed_hh_cmnty_cli` signal.
 (Click the "Code" button to see the Python code used to produce this example.)
 
+```{python, include=FALSE}
+import matplotlib.pyplot as plt # workaround for render problem due to weird interaction among reticulate, covidcast and matplotlib
```

Comment:
interesting fix!  howd you find it?!?  would it work to just change the order of the imports in the block below instead of including this extra hidden block?

also, code suggestion to remove extraneous comma:

```suggestion
```{python include=FALSE}
import matplotlib.pyplot as plt # workaround for render problem due to weird interaction among reticulate, covidcast and matplotlib
```

Reply:
The re-ordering works!

Reply:
nice!